esp-storage/src/common.rs

```diff
@@ -5,16 +5,24 @@ use crate::chip_specific;
 #[derive(Debug)]
 #[non_exhaustive]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
+/// Flash storage error.
```

Comment:
Just because you're touching this, this should also be clone, copy, partialeq, possibly eq